assets/javascripts/discourse/initializers/extend-for-translate-button.js

```diff
@@ -22,8 +23,9 @@ function initializeTranslation(api) {
       { before: ["search"] }
     );
   }
-
-  if (currentUser) {
+  if (siteSettings.experimental_topic_translation) {
+    api.renderInOutlet("topic-navigation", ShowOriginalContent);
+  } else {
```

Comment:
I think that we should still check `if(currentUser ||  SiteSetting.experimental_anon_language_switcher)`